ci/Dockerfile

```diff
@@ -34,7 +34,7 @@ ENV GOPATH=/opt/app-root/go \
     MAVEN_HOME="/opt/maven/${MAVEN_VERSION}" \
     GRADLE_HOME="/opt/gradle/${GRADLE_VERSION}" \
     SBT_HOME="/opt/sbt/${SBT_VERSION}" \
-    PYTHON_CMD=python3 \
+    PYTHON_CMD=/usr/bin/python${PYTHON_VERSION} \
```

Comment:
We may have to hard code the value or do this in a new ENV block. This is one of the known quirks of dockerfile.

Reply:
Yes understood, I will hard code the value.
FYI: I had created a new PR with the same change, I will add this there